<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasuresDockWidget</class>
 <widget class="QWidget" name="MeasuresDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>91</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Channel 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="channel1Table">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <row>
            <property name="text">
             <string>V</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>V Max</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>V Min</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Frequ.</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Nouvelle colonne</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Channel 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="channel2Table">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>49</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <row>
            <property name="text">
             <string>V</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>V Max</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>V Min</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Frequ.</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Nouvelle colonne</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Maths channel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="mathsTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>49</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <row>
            <property name="text">
             <string>Frequ.</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Phase</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>VMax</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Vmin</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Nouvelle colonne</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Vertical cursors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="cursor1Table">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>49</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <row>
            <property name="text">
             <string>Period.</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Frequ.</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Nouvelle colonne</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Horizontal cursors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="cursor2Table">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>151</width>
             <height>60</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>49</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>15</number>
           </attribute>
           <row>
            <property name="text">
             <string>V</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>V Max</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>V Min</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <row>
            <property name="text">
             <string>V Eff.</string>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Nouvelle colonne</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
